<launch>

  <arg name="spin_rate" default="50" />
  
	<node name="desperate_mond_node" pkg="desperate_housewife" type="desperate_mind" output="screen" >
    <param name="spin_rate" value="$(arg spin_rate)" />
  </node>

</launch>
